code/modules/pda/cart.dm

```diff
@@ -158,15 +158,23 @@
 		new /datum/data/pda/app/status_display
 	)
 
+/obj/item/cartridge/qm
+	name = "Space Parts & Space Vendors Cartridge DELUXE"
+	desc = "A data cartridge for portable microcomputers. Has supply records, MULEbot control and a status display controller."
+	icon_state = "cart-q"
+	programs = list(
+		new /datum/data/pda/app/supply,
+		new /datum/data/pda/app/mule_control,
+		new /datum/data/pda/app/status_display
+	)
+
```

Comment:
should this be purchasable in the p-tech vendor?

Reply:
Probably not since this has access to status displays